### DISCUSSION TITLE
Use library with Angular 9

### DISCUSSION THREAD

**Discussion by mauriciorzm** [katoid/angular-grid-layout#56] [Q&A] [answered]
Hello, is it possible to use this library with Angular 9? It was working well on Angular 12, but in 9 I tried it and got a bunch of errors like these:

![image](https://user-images.githubusercontent.com/17533479/176059854-67bc9d60-7bbb-4edb-8b90-b3f401248504.png)

**Comment by llorenspujol** [comment, marked as answer]
Check this: https://github.com/katoid/angular-grid-layout/issues/29#issuecomment-1017226309

You can use version 0.2.0, but be careful, some bug fixes or features are not done in this version, check the CHANGELOG.md to know more.

**Reply by mauriciorzm** [reply]
I will try that, thank you!
